pkg/ucp/frontend/controller/resourcegroups/util.go

```diff
@@ -215,10 +223,11 @@ func isOperationResourceType(id resources.ID) bool {
 		return true
 	}
 
-	// An older pattern is to use a child resource
+	// An older pattern is to use a child resource, it might also use the name "operations"
 	typeSegments := id.TypeSegments()
 	if len(typeSegments) >= 2 && (strings.EqualFold(typeSegments[len(typeSegments)-1].Type, "operationstatuses") ||
-		strings.EqualFold(typeSegments[len(typeSegments)-1].Type, "operationresults")) {
+		strings.EqualFold(typeSegments[len(typeSegments)-1].Type, "operationresults") ||
+		strings.EqualFold(typeSegments[len(typeSegments)-1].Type, "operations")) {
```

Comment:
Do we handle calls to /operations endpoint?

Reply:
I don't understand the question, can you be more specific? I think the comments explain this pretty well already.